_overviews/scala3-book/interacting-with-java.md

```diff
@@ -39,209 +39,277 @@ Note that the Java examples in this section assume that you’re using Java 11 o
 
 ## How to use Java collections in Scala
 
-When you’re writing Scala code and need to use a Java collection class, you _can_ just use the class as-is.
-However, if you want to use the class in a Scala `for` loop, or want to take advantage of the higher-order functions on the Scala collections classes, you’ll want to convert the Java collection to a Scala collection.
+When you’re writing Scala code and an API either requires or produces a Java collection class (from the `java.util` package), then it is valid to directly use or create the collection as you would in Java.
+
+However, for idiomatic usage in Scala, such as `for` loops over the collection, or to apply higher-order functions such as `map` and `filter`, you can create a proxy that behaves like a Scala collection.
 
 Here’s an example of how this works.
-Given this Java `ArrayList`:
+Given this API that returns `java.util.List[String]`:
 
+{% tabs foo-definition %}
+{% tab Java %}
 ```java
-// java
-public class JavaClass {
-  public static List<String> getStrings() {
-    return new ArrayList<String>(List.of("a", "b", "c"));
-  }
+public interface Foo {
+  static java.util.List<String> getStrings();
```

Comment:
I don’t think abstract static methods exist in Java.

Reply:
right, that makes sense. Really I only wanted to list the signature rather than show source code, not sure the best way

Reply:
I added in an implementation.